src/System.Private.ServiceModel/tests/Scenarios/Contract/Message/MessageContractTests.cs

```diff
@@ -15,7 +16,11 @@ public static void MessageContract_IsWrapped_True()
     {
         StringBuilder errorBuilder = new StringBuilder();
 
-        XmlDictionaryReader reader = MessageContractHelpers.SetupMessageContractTests(isWrapped: true);
+        MessageContractHelpers.IMessageContract clientProxy;
```

Comment:
Can I open a new issue for getting rid of the errorBuilders as these tests are existing tests?